storage/objects/objects_test.go

```diff
@@ -133,6 +134,13 @@ func TestObjects(t *testing.T) {
 		}
 	}
 
+	if err := copyFileArchivedGeneration(ioutil.Discard, bucketVersioning, bucketVersioning, object1, gen); err != nil {
```

Comment:
Two similar `bucketVersioning` args?

Reply:
Yes, to copy object inside the bucket. I saw in Java they did [this way](https://cloud.google.com/storage/docs/using-object-versioning#copy), but they didn't pass `dstBucket` in args. In c++, they copy from one bucket to another.
I'm not sure about keeping it in test, but it's comfortable 'cause we don't use `bucketVersioning` for other steps except disabling version and deleting, and other buckets we use.

Reply:
I mean, it looks like you're copying an object to the same bucket. Don't think it's very different from copy into another bucket, but who knows